root/components/home/About.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * @flow strict
+ * Copyright (C) 2022 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+// eslint-disable-next-line import/no-unresolved
+import {faCheckCircle} from '@fortawesome/free-solid-svg-icons';
+
+const About = (): React.Element<'section'> => (
+  <section className="p-4">
+    <div className="container">
+      <div className="row">
+        <div className="col-lg-6">
+          <h3 className="fs-1 fw-bolder">
+            {
+            // eslint-disable-next-line max-len
+              exp.l('About <span class="color-purple">Music</span><span class="color-orange">Brainz</span>')
+            }
+          </h3>
+          <p className="fs-4">
```

Comment:
I would just add `fs-4` to the parent div, since 3/4 children have it, and remove it from the 3 children.

Reply:
Done!